app/views/admin/index.scala.html

```diff
@@ -11,385 +11,426 @@
+                                <th>Today</th>
+                                <th>Yesterday</th>
                             </tr>
                             <tr>
-                                <td id="map-legend-curb-ramp" width="12px"></td>
-                                <td width="190px">Curb Ramp</td>
-                                <td width="12px"  align="center"><input type="checkbox" value="displaylabel" id="curbramp" checked="true" onclick="admin.updateVisibleMarkers();"></td>
-                                <td width="126px"align="center"><div id = "curb-ramp-slider" style="margin-top:3px"></div></td>
-                                <td width="60px" align= "center" ><span id="curb-ramp-severity-label">All</span></td>
-
+                                <th>Total Users</th>
+                                <th>@UserDAOImpl.countContributingUsers</th>
+                                <th>@UserDAOImpl.countTodayUsers</th>
+                                <th>@UserDAOImpl.countYesterdayUsers</th>
+                            </tr>
+                            <tr style="font-size: 90%;">
+                                <td style="padding-left: 30px">Registered</td>
+                                <td>@UserDAOImpl.countRegisteredUsers</td>
+                                <td>@UserDAOImpl.countRegisteredUsersVisitedToday</td>
+                                <td>@UserDAOImpl.countRegisteredUsersVisitedYesterday</td>
+                            </tr>
+                            <tr style="font-size: 90%;">
+                                <td style="padding-left: 30px">Anonymous</td>
+                                <td>@UserDAOImpl.countAnonymousUsers</td>
+                                <td>@UserDAOImpl.countAnonymousUsersVisitedToday</td>
+                                <td>@UserDAOImpl.countAnonymousUsersVisitedYesterday</td>
                             </tr>
                             <tr>
-                                <td id="map-legend-no-curb-ramp"></td>
-                                <td>Missing Curb Ramp</td>
-                                <td><input type="checkbox" value="displaylabel" id="missingcurbramp" checked="true" onclick="admin.updateVisibleMarkers();"></td>
-                                <td align="left"><div id = "missing-curb-ramp-slider" style="margin-top:3px"></div></td>
-                                <td align= "center"><span id="missing-curb-ramp-severity-label">All</span></td>
+                                <th>Total Labels</th>
+                                <th>@LabelTable.countLabels</th>
+                                <th>@LabelTable.countTodayLabels</th>
+                                <th>@LabelTable.countYesterdayLabels</th>
+                            </tr>
+                            <tr style="font-size: 90%;">
+                                <td style="padding-left: 30px">Curb Ramp
+                                </th>
```

Comment:
The closing tags are incorrect in this sub-section of "Activities".

<td>....</th>